src/inputs/Button/Button.stories.tsx

```diff
@@ -1,23 +1,33 @@
-// Button.stories.tsx
 import { Meta, Story } from "@storybook/react";
-import * as React from "react";
-import { Button as MyButton, ButtonProps } from "./Button";
+import { Button } from "./Button";
+import React from "react";
+import { Plus } from "../../icons";
 
 export default {
-  component: MyButton,
+  component: Button,
   argTypes: {
-    variant: {
-      options: ["primary", "secondary"],
-      control: { type: "select" },
+    icon: {
+      options: {
+        none: null,
+        Settings: <Plus />,
+      },
     },
   },
 } as Meta;
 
-export const Button: Story = (args) => (
-  <MyButton {...(args as ButtonProps)}>Click me!</MyButton>
+export const Default: Story = (args) => (
+  <Button
+    icon={<Plus />}
```

Comment:
**nitpick:** This icon looks definitely too big. Let's scale it down. + It does not look properly without icon (text is not centered):

<img width="364" alt="Zrzut ekranu 2023-04-7 o 18 51 27" src="https://user-images.githubusercontent.com/22893188/230646706-43ca3245-e826-4c4f-8b21-a72173e67d7c.png">